docs/other-sdks.md

```diff
@@ -0,0 +1,17 @@
+---
+id: other-sdks
+title: Other SDKs
+sidebar_label: Other SDKs
+---
+
+There is demand for other Temporal SDKs other than Go and Java. The user community has taken initiative to build some of these, but please note that they are not officially supported or endorsed by Temporal unless explicitly mentioned:
+
+- [Ruby SDK by Coinbase](https://github.com/coinbase/temporal-ruby)
+- Python SDK - please get in touch with us
+- Swift SDK - please get in touch with us
+- C#/dotNet SDK - please get in touch with us
+- Node.js SDK - please get in touch with us
+- Rust SDK - please get in touch with us
+- C++ SDK - please get in touch with us
+
+We are very excited by the interest of the community, but please don't go too far without getting in touch with us! Collaboration via open source is something we strongly believe in, but we also care about the long term developer experience and therefore likely will want to take on maintenance leadership of SDKs in key languages.
```

Comment:
I think this message could be more focused. To me it seems like the key message we want to communicate here is that we want people to work with us. I am not sure if we want to say that we want to take ownership directly here...Consider:

If you are interested in using an unofficial SDK, we recommend that you reach out to any of us at Temporal. We are very excited by the community's enthusiasm and strongly believe in collaborating on open source projects. Our goal is to make sure there is maintenance leadership in place to care for the developer experience over the long term.

Reply:
```suggestion
If you are interested in using an unofficial SDK, we recommend that you reach out to any of us at Temporal. We are very excited by the community's enthusiasm and strongly believe in collaborating on open source projects. Our goal is to make sure there is maintenance leadership in place to care for the developer experience over the long term.
```